packages/core/src/provider/schema.ts

```diff
@@ -9,22 +9,22 @@ export const SmartAccountProviderOptsSchema = z.object({
   /**
    * The maximum number of times to try fetching a transaction receipt before giving up (default: 5)
    */
-  txMaxRetries: z.number().optional().default(5),
+  txMaxRetries: z.number().optional(),
 
   /**
    * The interval in milliseconds to wait between retries while waiting for tx receipts (default: 2_000)
    */
-  txRetryIntervalMs: z.number().optional().default(2_000),
+  txRetryIntervalMs: z.number().optional(),
 
   /**
    * The mulitplier on interval length to wait between retries while waiting for tx receipts (default: 1.5)
    */
-  txRetryMulitplier: z.number().optional().default(1.5),
+  txRetryMulitplier: z.number().optional(),
 
   /**
    * used when computing the fees for a user operation (default: 100_000_000n)
    */
-  minPriorityFeePerBid: z.bigint().optional().default(100_000_000n),
```

Comment:
why defaults removed? on the comment, it still says defaults

Reply:
![image](https://github.com/alchemyplatform/aa-sdk/assets/43521356/e942834c-9bab-44c9-993f-778f6f4db939)

Yeah sadly the default made these fields required even though they're optional.

Defaults are still set in the constructor.

Reply:
Yeah had the same with entrypoint